tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn_with_text.py

```diff
@@ -33,10 +33,6 @@ def parameters(cls):
                 description="Name of output layer with information about classes.",
                 optional=False
             ),
-            'scores_out': StringField(
```

Comment:
please keep code for support 0002. We still have it in quantization scope...

Reply:
just make scores_out is optional

Reply:
fixed